askomics/libaskomics/rdfdb/QueryLauncher.py

```diff
@@ -98,6 +98,10 @@ def parse_results(self, json_res):
 
     def process_query(self, query):
         json_query = self.execute_query(query, log_raw_results=False)
+
+        if 'CLEAR GRAPH' in query:
```

Comment:
I'm not very happy with this if block, what happens if you go on with the parse_results method?